### PR TITLE
More cleaning up

### PR DESCRIPTION
**Pull request by thutterer**

### CODE REVIEW

File: app/views/links/_form.html.haml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    .col-md-offset-3
+  .form-group
+    = f.label :url, class: "control-label col-md-2"
+    .col-md-7
+      = f.text_field :url, class: "form-control", required: true
+    .col-md-offset-3
+  .form-group
+    .col-md-offset-2.col-md-7
+      = f.submit 'Save', class: "btn btn-primary"
+      = link_to 'Cancel', users_path, class: "btn btn-default"
```
Comment (NadjaW) [inline app/views/links/_form.html.haml]: users_path is wrong here ;)

File: app/controllers/links_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    else
+      render action: :edit
+    end
+  end
+
+  def destroy
+    @link = Link.find(params[:id])
+    @link.destroy
+
+    flash[:success] = 'Link deleted.'
```
Comment (NadjaW) [inline app/controllers/links_controller.rb]: 'Link deleted' flash message is not aligned with the links table. Links table is smaller than the message

Comment (thutterer) [inline app/controllers/links_controller.rb]: layout of flash messages is broken in the hole app... it's on my list for one of the next clean-up days :)

**Comment by NadjaW** [comment]
lgtm

**Pull request merged**